services/github/schemas/v1/taskcluster-render-yaml-output.yml

```diff
@@ -0,0 +1,22 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "Render .taskcluster.yml output"
+description: Rendered .taskcluster.yml output.
+type: object
+properties:
+  tasks:
+    type: array
+    description: Rendered task
+    uniqueItems: false
+    items:
+      type: object
+      additionalProperties: true
+  scopes:
```

Comment:
is this not already in all of the rendered tasks?

Reply:
those are different scopes! but it is a very good observation. 
I should add a description that those scopes are used for `queueClient` when github service is creating those tasks in the first place. But tasks, themselves, have other scopes inside their definition body

Reply:
improved documentation